apps/style/craft/style.scss

```diff
@@ -400,7 +405,7 @@ $height-to-width: 434 / 477;
 
 #craft-popup-connect {
   @include reset-dialog;
-  background-color: #3C3C3C;
+  background-color: $almost-very-dark-gray;
```

Comment:
With names like these, are we really making things that much clearer by using english vs. hex codes?